schedule.md

```diff
@@ -6,19 +6,41 @@ Always be commiting. Deploy early and often.
 
 Here's a rough sketch of what you should do and in what order:
 
-1.  [ ] Sketch some rough wireframes for how the front end will look and act.
-1.  [ ] Write out some user stories for the app. User stories are short of how a
-    user interacts with your app, and follows the format "As a `<role>`, I want
-    to `<do something>`, so that `<some goal>`.
-1.  [ ] Model the entities in your app. Draw a diagram. Use your wireframes and
-    user stories to drive you modeling process by asking "What 'things' are a
-    user interacting with?"
```

Comment:
I want things done in the is order. This is the natural flow of the planning process, and the explanation here is something the developers should read (at least) if not discuss amongst themselves during an exercise.

People visualize applications first. Then by "clicking" through the frames, they identify page flow and therefore user stories. This natural drafting culminates in identified nouns and verbs to reify in the system via entities and relationships.